docs/user-guide/configuration.md

```diff
@@ -355,6 +355,28 @@ extra:
 
 ## Preview controls
 
+## Live Reloading
+
+### watch
+
+Determines additional directories to watch when running `mkdocs serve`.
+Configuration is a YAML list.
+
+```yaml
+watch:
+- directory_a
+- directory_b
+```
+
+Allows a custom default to be set without the need to pass it through the `-w`/`--watch`
+option every time the `mkdocs serve` command is called.
+
+!!! Note
+
+    The paths provided via the configuration file are relative to the configuration file.
+
+    The paths provided via the `-w`/`--watch` CLI parameters are not. 
```

Comment:
Bit late to the party, but what exactly would it then be? What would it's "position" be based of? Where the command is executed? That should've been mentioned here IMO, so maybe in a future PR/Commit could this be changed?